.aliases

```diff
@@ -52,7 +52,12 @@ alias week='date +%V'
 alias update='sudo softwareupdate -i -a; brew update; brew upgrade; brew cleanup; npm install npm -g; npm update -g; sudo gem update --system; sudo gem update; sudo gem cleanup'
 
 # Google Chrome
-alias chrome='/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome'
+if [ "$(uname)" = "Linux" ] ; then
+	alias chrome='/opt/google/chrome/google-chrome'
+else
+	alias chrome='/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome'
+fi
```

Comment:
Instead of checking `uname` output and assuming different paths, it seems better to check if the path exists.

Reply:
@mathiasbynens 👍, I have updated to check if the file exists instead before aliasing chrome in Linux.